opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -377,10 +379,15 @@ def _format_attribute_value(value: types.AttributeValue) -> AttributeValue:
     elif isinstance(value, float):
         value_type = "string_value"
         value = _get_truncatable_str_object("{:0.4f}".format(value), 256)
+    elif isinstance(value, collections.Sequence):
```

Comment:
Did OT say specifically that only `Sequence` is allowed, or can it be any iterable (this will not pass for unordered iterables like set and dict)?

Reply:
OT SPec
```
AttributeValue = Union[
    str,
    bool,
    int,
    float,
    Sequence[str],
    Sequence[bool],
    Sequence[int],
    Sequence[float],
]
```
So the value shouldn't be a set or dict.